keps/1282-requeue-strategy/README.md

```diff
@@ -0,0 +1,306 @@
+# KEP-1282: Requeue Strategy
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [API Changes](#api-changes)
+  - [Changes to Queue Sorting](#changes-to-queue-sorting)
+    - [Existing Sorting](#existing-sorting)
+    - [Proposed Sorting](#proposed-sorting)
+    - [Preemption/Preemptor Flapping](#preemptionpreemptor-flapping)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+Introduce new options that allow administrators to configure how Workloads are placed back in the queue after being after being evicted.
+
+## Motivation
+
+### Goals
+
+Allow administrators to configure requeuing behavior based on the reason the Workload is being requeued.
+
+### Non-Goals
+
+## Proposal
+
+Make queue placement after preemption configurable at the level of Kueue configuration.
```

Comment:
Why not per ClusterQueue? I guess it makes a lot of sense for WaitForPodsReady, because it's also a Kueue level configuration.
But I'm not sure about other eviction options. I guess it doesn't have to be the same for all.

Reply:
As we may divide the resource to different teams by ClusterQueue, making it as the ClusterQueue level configuration can be beneficial in accommodating various scenarios.

Reply:
If it is implemented at the ClusterQueue level, what do you think is the best way to handle the sorting of heads across ClusterQueues?

Reply:
Kept in ConfigMap